.eslintignore

```diff
@@ -1,2 +1,3 @@
 node_modules
 /docs
+*.min.js
```

Comment:
Now that this is hosted on jsDelivr, could you remove this line?

Reply:
Done! Thank you!